apps/website/src/pages/docs/tooltip.mdx

```diff
@@ -9,18 +9,27 @@ group: core
 
 import PropsTable from '~/components/PropsTable.astro';
 import LiveExample from '~/components/LiveExample.astro';
-import Placeholder from '~/components/Placeholder.astro';
 import * as AllExamples from '~/examples';
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='core-tooltip' />
-
 <LiveExample src='Tooltip.main.tsx' truncate={false}>
   <AllExamples.TooltipMainExample client:load />
 </LiveExample>
 
-A tooltip is a small container that typically appears above the element you are hovering with your cursor. The tooltip displays a short, helpful text message. Tooltips should never be used to display critical information, or as a means to navigate an interface. Because mobile devices do not have a hover state, use caution when using tooltips in desktop apps that have a mobile counterpart. There is no determined way of replacing tooltips in mobile devices. In some cases, a single short tap can display a tooltip on a touch display.
+A tooltip is a small container that typically appears above the element you are hovering with your cursor with a short, helpful text message.
+
+## Usage
+
+You should use up to 256 characters. Content other than text such as pictures, tables, etc, are not recommended. Mobile devices does not have a hover state, caution when using tooltips in desktop apps that have a mobile counterpart. There is no determined way of replacing tooltips in mobile devices.
+
+Examples of many ways tooltip can be use in [Avatar](avatar#overflow) group, [Stepper](stepper#tooltip), and [Slider](slider#tooltips).
+
+It is not recommended to use tooltips to display critical information, or as a means to navigate an interface. They are necessary only when supplemental information is required. Functionality of the product should NOT be dependent on the user reading tooltips.
+
+Tooltips are enabled by default but can be turned off.
+
+Tooltips can also be customized such as changing the placement or the information that is displayed within it.
```

Comment:
I think you can add a demo example here, like the ones in Storybook, probably not all of them, and then list the possible values for `placement` like we do in Avatar doc?

![image](https://user-images.githubusercontent.com/22462084/235498432-ea186c4c-8fee-4f6a-83b2-466265822467.png)

I think you could also have this in its own section called Placement like in Alert doc as well
![image](https://user-images.githubusercontent.com/22462084/235498637-76f43e00-8ee6-461e-80c0-61990f7211d3.png)